controllers/external/util.go

```diff
@@ -38,6 +38,10 @@ const (
 
 // Get uses the client and reference to get an external, unstructured object.
 func Get(ctx context.Context, c client.Client, ref *corev1.ObjectReference, namespace string) (*unstructured.Unstructured, error) {
+	if ref == nil {
+		return nil, errors.Errorf("ref cannot be nil")
```

Comment:
Super happy to get some inspo for a better error message :D

Reply:
`external object ref cannot be nil`